helloworld/hello_world.md

```diff
@@ -139,235 +138,232 @@ while True:
 ```bash
 cd /root/
 git clone https://gitee.com/InternLM/Tutorial -b camp2
 # git clone https://github.com/InternLM/Tutorial -b camp2
 cd /root/Tutorial
 ```
 
-### 3.3 **下载运行 Chat-八戒 Demo**
+### 3.3 **3.3 Download Model and Run Bajie-Chat Demo**
 
-在 `Web IDE` 中执行 `bajie_download.py`：
+Execute `bajie_download.py` in `Web IDE` interface：
 
 ```bash
 python /root/Tutorial/helloworld/bajie_download.py
 ```
 
-待程序下载完成后，输入运行命令：
+After the model download program is completed, execute the following command:
 
 ```bash
 streamlit run /root/Tutorial/helloworld/bajie_chat.py --server.address 127.0.0.1 --server.port 6006
 ```
 
-待程序运行的同时，参考章节 3.3 部分对端口环境配置本地 `PowerShell` 。使用快捷键组合 `Windows + R`（Windows 即开始菜单键）打开指令界面，并输入命令，按下回车键。（Mac 用户打开终端即可）
+While waiting for the program to run, you can try to configuring the port forwarding setting in local `PowerShell`. Use the shortcut key combination `Windows + R` (the Windows key is the Start menu key) to open the command interface, then type the command PowerShell and press Enter.(Mac users just need to simply open the terminal)
 
 ![alt text](images/img-8.png)
 
-打开 PowerShell 后，先查询端口，再根据端口键入命令 （例如图中端口示例为 38374）：
+After opening PowerShell, query the port number, and then enter the command according to the port number (for example, the port example shown in the figure is 38961):
 
 ![alt text](images/img-A.png)
 
 ```bash
-# 从本地使用 ssh 连接 studio 端口
-# 将下方端口号 38374 替换成自己的端口号
-ssh -CNg -L 6006:127.0.0.1:6006 root@ssh.intern-ai.org.cn -p 38374
+# connect to the InternStudio port from the local machine via SSH
+# Replace the port number 38961 below with your own InternStudio cloud machine port number
+ssh -CNg -L 6006:127.0.0.1:6006 root@ssh.intern-ai.org.cn -p 38961
 ```
 
-再复制下方的密码，输入到 `password` 中，直接回车：
+Copy the `password` shown in the password field, paste it into PowerShell and press Enter:
 
 ![alt text](images/img-B.png)
 
-最终保持在如下效果即可：
+The final result should be as follows:
 
 ![alt text](images/img-C.png)
 
-打开 [http://127.0.0.1:6006](http://127.0.0.1:6006) 后，等待加载完成即可进行对话，键入内容示例如下：
+Open the following link: [http://127.0.0.1:6006](http://127.0.0.1:6006). Once the model loading is complete, you can proceed with the conversation. Type in content example as follows:
 
-    你好，请自我介绍
+    Hi, please introduce yourself.
 
-效果图如下：
+The result is as follows：
 
 ![alt text](images/img-D.png)
 
-##  4 **实战：使用 `Lagent` 运行 `InternLM2-Chat-7B` 模型（开启 30% A100 权限后才可开启此章节）**
+##  4 **Run a `Demo` of agent using `Lagent` with the `InternLM2-Chat-7B` Model (Prerequisite: This chapter requires obtaining permission of 30% A100)**
 
-### 4.1 **初步介绍 Lagent 相关知识**
-Lagent 是一个轻量级、开源的基于大语言模型的智能体（agent）框架，支持用户快速地将一个大语言模型转变为多种类型的智能体，并提供了一些典型工具为大语言模型赋能。它的整个框架图如下:
+### 4.1 **Brief Introduction to Lagent**
+Lagent is an open source LLM agent framework, which enables people to efficiently turn a large language model to agent. It also provides some typical tools to enlighten the ability of LLM, and the whole framework is shown below:
```

Comment:
Lagent is an open source LLM agent framework, which enables people to efficiently turn a large language model to agent. ***It also provides some typical tools to enhance the capabilities of LLM***, and the whole framework is shown below: